addons/website_forum/static/src/css/website_forum.css

```diff
@@ -183,3 +183,6 @@ button.btn-link.text-muted {
 .o_forum_user_bio .o_forum_tooltip_header {
   min-height: 75px;
 }
+.oe_no_empty {
+  word-wrap: break-word;
+}
```

Comment:
Could you explain this to me ?

Reply:
@qsm-odoo it was creating problem, see video https://drive.google.com/file/d/0B8SK9KKMbAOjVDhGQ2JJblI1QkU/view
 but this also has been fixed so we can remove this too :p